src/ifcparse/Ifc2x3enum.h

```diff
@@ -29,7 +29,7 @@
 #ifndef IFC2X3ENUM_H
 #define IFC2X3ENUM_H
 
-#include "../ifcparse/IFC_PARSE_API.h"
+#include "../ifcparse/ifc_parse_api.h"
```

Comment:
Already fixed in the right location, but an accidental replace-in-files had made in uppercase in the generated sources, https://github.com/Tridify/IfcOpenShell/blob/ifcgeom_dll/src/ifcexpressparser/templates.py